docs/dev/table/streaming/temporal_tables.md

```diff
@@ -36,6 +36,14 @@ For the changing dimension table, Flink allows for accessing the content of the
 Motivation
 ----------
```

Comment:
temporal tables got a lot of new features in 1.12. Please rebase, unfortunately, a lot of what you did is no longer relevant.